### PR TITLE
[ Fix ] : Other languages not usable in whiteboard

### PR DESCRIPTION
**Pull request by Sumitwarrior7**

### CODE REVIEW

**Comment by Sumitwarrior7** [comment]
> @Sumitwarrior7 can you attach a mini video of working feature ?

ok bhaiya

**Comment by Sumitwarrior7** [comment]
https://github.com/RocketChat/Apps.Whiteboard/assets/108853577/02533532-bca6-4b95-b4e3-d60e2c9f815e

**Comment by CulturalProfessor** [comment]
> langugae_change_contr1.mp4

woah really good for your 1st pr :rocket:

File: client/src/i18n.ts [hunk 1]
```diff
@@ -86,18 +185,22 @@ let currentLangData = {};
...
-        /* webpackChunkName: "locales/[request]" */ `./locales/${currentLang.code}.json`
-      );
+      currentLangData = langImportsMap[currentLang.code];
+      // currentLangData = await import(
+      //   /* webpackChunkName: "locales/[request]" */ `./locales/${currentLang.code}.json`
+      // );
     } catch (error: any) {
-      console.error(`Failed to load language ${lang.code}:`, error.message);
+      console.error(
+        `language load karne me dikkat aa rahi hai  ${lang.code}:`,
```
[the remark this answers is not in the document]
Comment (Sumitwarrior7) [reply client/src/i18n.ts]: bhaiya removed this log

**Comment by CulturalProfessor** [comment]
LGTM ! I think we can merge this now :tada:

**Pull request merged**